.github/ISSUE_TEMPLATE/release.md

```diff
@@ -16,7 +16,30 @@ This is a checklist for releases. This is filled in by both the releaser and the
 #### Update Documentation
 
 - [ ] Update the [documentation version](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/config/_default/config.toml#L28) to match the current minor, if necessary (`v3.${minor}`).
-- [ ] Run `mage ttiProto:hugoData` from [TheThingsIndustries/lorawan-stack](https://github.com/TheThingsIndustries/lorawan-stack), and copy the generated files to the [TheThingsIndustries/lorawan-stack-docs](https://github.com/TheThingsIndustries/lorawan-stack-docs).
+- [ ] To generate documentation, create a clone of [TheThingsIndustries/lorawan-stack](https://github.com/TheThingsIndustries/lorawan-stack), and **checkout the correct branch**, i.e the current minor branch.
+- [ ] To generate API documentation, run the following from within the clone of [TheThingsIndustries/lorawan-stack](https://github.com/TheThingsIndustries/lorawan-stack): 
+
+```
+  tools/bin/mage ttiProto:hugoData
+```
+
+- [ ] Copy generated files to `lorawan-stack-docs` with the following command:
+
+```
+  cp -r doc/data/api /path/to/lorawan-stack-docs/doc/data/
+```
+
+- [ ] To generate and export CLI documentation from within the clone of [TheThingsIndustries/lorawan-stack](https://github.com/TheThingsIndustries/lorawan-stack), run the following command:
+
+```
+  go run -tags tti ./cmd/tti-lw-cli gen-yaml-doc -o /path/to/lorawan-stack-docs/doc/data/
```

Comment:
You should `go install` (or `go build`) `-tags tti ./cmd/tti-lw-cli` and then run `HOME='$HOME' tti-lw-cli gen-yaml-doc`, otherwise it will expand your home directory in the output.